linode/manager
# fix: VPC Detail page document title

## Description 📝
Fix the VPC Detail page document title to include the VPC label.

Also, adjust the formatting of VPC to-do's in code comments to be `@TODO VPC: [...]`

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-09-13 at 12 40 53 PM](https://github.com/linode/manager/assets/114682940/af96cfd5-e878-4702-b673-006b03c8df5e) | ![Screenshot 2023-09-13 at 12 41 06 PM](https://github.com/linode/manager/assets/114682940/88df2270-a063-4c3b-ab7e-4b71c198c842) |

## How to test 🧪
Check VPC Detail pages and confirm that the first segment in the document title is the VPC's label (as opposed to saying `VPC | VPC | Akamai Cloud Manager` outside of this branch)
